api/db-cdr/generate-cdr/cloudsql-import-bucket.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+# Imports all sql and csv files in the bucket to the database.
+# IF --create-dump-file is specified, it is ran first
```

Comment:
I'm not following - what is the relationship between --create-dump-file and the rest of this script? It seems almost like an unrelated action